docs/storage/petalibrary/onedrive.md

```diff
@@ -0,0 +1,88 @@
+### Copy data from OneDrive to PetaLibrary
+
+### Using Globus
+
+#### Step 1: Log into the Globus Web App
+Visit https://app.globus.org and log in with your Colorado.edu credentials
+
+![Globus Login](Globus_Login.png)
+
+#### Step 2: Open your OneDrive (right pane)
+In the Collection search (right pane) enter: OneDrive CU Boulder - this should show your OneDrive contents
```

Comment:
--> (capitalize Search and add comma) In the Collection Search (right pane), enter: